scale-testing.mk

```diff
@@ -0,0 +1,32 @@
+# run the following from the project root:
+# make -f scale-testing.mk <make rule>
+
+# AWS variables for cloud hosted k8s testing
+AWS_REGION ?= us-east-2
+EKS_K8S_VERSION ?= 1.26
+
+# directories for cloud hosted k8s infrastructure for running tests
+# root directory for all integration tests
+TF_EKS_SRC_DIR ?= $(INTEGRATION_TEST_ROOT)/infra/scale-testing
+TF_EKS_STATE_DIR ?= $(TF_EKS_SRC_DIR)/state
+
+include ./Makefile
+
+##@ EKS
+
+.PHONY: create-eks
+create-eks: ## Create a new EKS cluster
+	@mkdir -p $(TF_EKS_STATE_DIR)
+	rm -f $(TF_EKS_STATE_DIR)/*.tf
+	cp -v $(TF_EKS_SRC_DIR)/*.tf $(TF_EKS_STATE_DIR)/.
+	$(TERRAFORM) -chdir=$(TF_EKS_STATE_DIR) init -upgrade
+	$(TERRAFORM) -chdir=$(TF_EKS_STATE_DIR) apply -auto-approve \
+		-var region=$(AWS_REGION) \
+		-var kubernetes_version=$(EKS_K8S_VERSION) || exit 1
+	rm -f $(TF_EKS_STATE_DIR)/*.tfvars
```

Comment:
Good idea, thank you! I added it

Reply:
I am having some issues with `destroy-eks` target tho. I am getting insufficient permissions to destroy it. Any suggestions on how to resolve, considering that we can't update permissions on our doormat account

Reply:
I've been having issues with `destroy-eks` failing too, though it seems to get stuck on deleting k8s resources or one of the helm charts. I don't think it's actually a AWS permissions issue, since removing any of the `kubernetes_*` or `helm_release` resources from the terraform.tfstate file allows it to proceed correctly for me.

Reply:
Is there a way where we can set make targets to remove those resources before destroying the cluster? I am having issues with the destroy targets :/

Reply:
Splitting them into two runs resolves this issue. I can now destroy it without any issues.